index.html

```diff
@@ -4499,16 +4499,17 @@ <h2>The Role of Human-Friendly Identifiers</h2>
       <h2>DIDs as Enhanced URNs</h2>
 
       <p>
-If desired by a <a>DID controller</a>, a <a>DID</a> is capable of acting
-as an enhanced Uniform Resource Name (URN) as defined by [[RFC8141]], i.e.,
-"a persistent, location-independent resource identifier". <a>DIDs</a> used in
-this way provide a cryptographically secure, location-independent identifier for
-a digital resource, while also providing metadata that enables retrieval.
-Because of the indirection between the <a>DID document</a> and the <a>DID</a>
-itself, the <a>DID controller</a> can adjust the actual location of the resource
-&mdash; or even provide the resource directly &mdash; without adjusting the
-<a>DID</a>. <a>DIDs</a> of this type can definitively verify that the resource
-retrieved is, in fact, the resource identified.
+If desired by a <a>DID controller</a>, a <a>DID</a> is capable of acting as a
+persistent, location-independent resource identifier. These sorts of identifiers
```

Comment:
In other words, 
```suggestion
If desired by a <a>DID controller</a>, <a>DIDs</a> and <a>DID URLs</a> are
capable of acting as persistent, location-independent resource identifiers.
These sorts of identifiers
```
?  Yes, I think so.  Or perhaps (not as a suggestion, since github can only handle one at a time) --
```
If desired by a <a>DID controller</a>, a <a>DID</a> or a <a>DID URL</a> is
capable of acting as a persistent, location-independent resource identifier.
These sorts of identifiers
```